portal/static/js/main.js

```diff
@@ -1263,18 +1269,33 @@ var fillContent = {
                 };
 
                 arrTypes.forEach(function(type) {
-                    if (typeInTous(type)) {
+                    if (typeInTous(type, "active")) {
                         item_found++;
                     };
                 });
 
-                if (item_found == arrTypes.length) {
+                var arrReconsent = $.grep(arrTypes, function(type) {
+                    return typeInTous(type, "inactive");
+                });
+
+                /*
+                 *  note display of checked checkbox when re-consenting is controlled by css
+                 */
+                if (arrReconsent.length > 0) {
+                    self.attr("data-reconsent", "true");
+                    if (!setReconsentDisplay) {
+                        $(this).closest("#termsCheckbox").attr("data-reconsent", "true");
+                        setReconsentDisplay = true;
+                    }
+                }
+
+                if (item_found !== 0 && (item_found == arrTypes.length)) {
```

Comment:
Should that be `item_found > 0` ? I assume the type-checking will be handled by the second condition...